README.md

```diff
@@ -34,9 +34,32 @@ $ cp .env.template .env  # (first time only)
 The `.env` file is used by flask to set environment variables when running `flask run`. This enables things like development mode (which also enables features like hot reloading when you make a file change). There's also a [SECRET_KEY](https://flask.palletsprojects.com/en/1.1.x/config/#SECRET_KEY) variable which is used to encrypt the flask session cookie.
 
 
-Update the three trello vars in the .env file to include the correct board id, API key and API token.
+Update the three trello vars in the .env file to include the correct board id, API key and API token you can get these by clicking the following link https://trello.com/app-key
 
 
+## Testing the code works as expected
+
+To test the app is working as expected there are two sets of tests.
+Unit tests are to check the code with a static data set, the data file is created to run against 3 tests
+Intergratio tests are to check the pages are running correctly with stub data
+
+to run these tests: 
+*Press control shift P in VSCode and find Python:Configure tests
+*Select Pytest then select the todo_app folder
+*Select the test window (dispalyed as a science beaker)
+*Run all tests under the test explorer
+*Also you can run these via the command line by typing "Poetry run Pytest"  
+
+
+## Running in docker 
+
+To build the to-do app in Docker run the following two commands depending on env you wish to run
+docker build --target development --tag todo-app:dev .
+docker build --target production --tag todo-app:prod .
+
+To run the APP in Docker run either of the two commands depending on env you wish to run
+docker run --env-file ./.env -p 3500:80 --mount type=bind,source="$(pwd)"/todo_app,target=/app/todo_app todo-app:dev
+docker run --env-file ./.env -p 3500:80 --mount type=bind,source="$(pwd)"/todo_app,target=/app/todo_app todo-app:prod
```

Comment:
You should also include the link to the docker registry image. As per the submission checklist **"Please include a link in your README"**